endtoend/components/slasher_simulator.go

```diff
@@ -0,0 +1,43 @@
+package components
+
+import (
+	"fmt"
+	"os/exec"
+	"strings"
+	"testing"
+
+	"github.com/bazelbuild/rules_go/go/tools/bazel"
+	"github.com/prysmaticlabs/prysm/endtoend/helpers"
+	e2e "github.com/prysmaticlabs/prysm/endtoend/params"
+)
+
+// StartSlasherSimulator runs a beacon chain client with the slasher simulator enabled to simulate slashing detection.
+func StartSlasherSimulator(t *testing.T) {
+	binaryPath, found := bazel.FindBinary("cmd/beacon-chain", "beacon-chain")
+	if !found {
+		t.Log(binaryPath)
+		t.Fatal("Beacon chain binary not found")
```

Comment:
Edit

Reply:
It's accurate, it uses the beacon chain binary.